gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -184,9 +187,12 @@ ROOT::Experimental::TWebWindowsManager::~TWebWindowsManager()
 
 bool ROOT::Experimental::TWebWindowsManager::CreateHttpServer(bool with_http)
```

Comment:
"with_http" is very surprising. What else? It's like `CreateHttpServer(bool server, bool http)` ;-) I guess this is http versus https? Then please either create a second function (`CreateHttpsServer`) or have an enum `kHttp, kHttps`.

Reply:
> "with_http" is very surprising. What else?

Not always THttpServer used with real HTTP/HTTPS through `civetweb`. 
THttpServer can serve requests locally - one just creates THttpCallArg instance and submits it. This is how CEF and QWebEngine are working.

Reply:
Well but then my point stands, the function is names `CreateHttpServer()` and you tell me that it doesn't create an http server, for `server == false`. That's bad :-) I am sure you can come up with a better name, or split the functionality into `ReallyCreateHttpServer()` ;-) and `DoTheStuffThatDoesNotActuallyCreateAnHttpServer()`?

Reply:
Ok, I rename function - now it called CreateServer()